instrumentation/aerospike-client/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/aerospike/v7_0/AerospikeClientInstrumentationModule.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.aerospike.v7_0;
+
+import static io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers.hasClassesNamed;
+import static java.util.Arrays.asList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.List;
+import net.bytebuddy.matcher.ElementMatcher;
+
+@AutoService(InstrumentationModule.class)
+public class AerospikeClientInstrumentationModule extends InstrumentationModule {
+
+  public AerospikeClientInstrumentationModule() {
+    super("aerospike-client");
+  }
+
+  @Override
+  public ElementMatcher.Junction<ClassLoader> classLoaderMatcher() {
+    return hasClassesNamed("com.aerospike.client.IAerospikeClient")
```

Comment:
`hasClassesNamed` accepts an array of strings so you could have one call to `hasClassesNamed` that lists all the classes. Usually you'd use `hasClassesNamed` here to avoid running the instrumentation on old version of library and here you'd test that a class that was added in the new version is present. Looking at the list of classes makes me wonder why you are doing this and whether this is needed at all.